apps/dashboard/src/components/variable/variable-list.tsx

```diff
@@ -148,22 +150,43 @@ const VariableListItem = ({
   hoveredOptionIndex,
   setHoveredOptionIndex,
   onSelect,
+  context = 'variables',
 }: {
   option: VariablesListProps['options'][number];
   index: number;
   selectedValue?: string;
   hoveredOptionIndex: number;
   setHoveredOptionIndex: (index: number) => void;
   onSelect: (value: string) => void;
+  context?: 'variables' | 'translations';
 }) => {
   const hasPreview = !!option.preview;
+  const isHovered = hoveredOptionIndex === index;
+  const timeoutRef = useRef<NodeJS.Timeout | null>(null);
+
+  const handleMouseLeave = () => {
+    // Small delay to allow moving to tooltip
+    timeoutRef.current = setTimeout(() => {
+      setHoveredOptionIndex(-1);
+    }, 150);
+  };
+
+  const handleMouseEnter = () => {
+    if (timeoutRef.current) {
+      clearTimeout(timeoutRef.current);
+      timeoutRef.current = null;
+    }
+
+    setHoveredOptionIndex(index);
+  };
```

Comment:
Is there any other way of handling this maybe? Usually those mouseleave things are creating troubles down the read. What is the issue we try to solve with this here?